types/rule_match.go

```diff
@@ -35,6 +35,11 @@ type MatchedRule interface {
 	TransactionID() string
 	// Disruptive is whether this rule will block the request
 	Disruptive() bool
+	// // Log is whether this rule is marked as expected to be logged
```

Comment:
```suggestion
	// Log is whether this rule is marked as expected to be logged
```

Reply:
Let's remove the whole change in this file, it looks like the best it can do is provide confusion rather than clarity. If there's no public method, there's no public method